ozpricechecker/pricefinderapp/tests/models/test_product.py

```diff
@@ -18,6 +20,20 @@ def test_str_no_name(self):
     def test_str_name(self):
         product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
         self.assertEqual(str(product), 'Woolworths - Super Tomatoes')
+    
+    def test_latest_prod_price_date(self):
```

Comment:
It might be easier to maintain if we separate the tests for the 2 properties and have at least a test for each

Reply:
Refactor later